compiler/catala_utils/cli.ml

```diff
@@ -58,6 +60,7 @@ let backend_option_to_string = function
 let backend_option_of_string backend =
   match String.lowercase_ascii backend with
   | "interpret" -> `Interpret
+  | "interpret_lcalc" -> `Interpret_Lcalc
```

Comment:
This new backend should be documented in the `info` variable in this file (around line ~300).

Reply:
I've added a docstring. Can you confirm it is ok?